24_find_lab/findlab.txt

```diff
@@ -1,24 +1,72 @@
 1. Write command lines to find using "find":
 
-  -. All files owned by yourself in /tmp folder
+  
+-. All files owned by yourself in /tmp folder
 
-  -. All files and directories not owned by you in /tmp folder
+  
+I typed find /tmp/ -user yulia and mostly receive permission messages

```

Comment:
That's a good thing, as you probably don't have access to many subdirectories there. You can suppress these errors using: `find /tmp/ -user yulia 2> /dev/null`

Also you can use the more generic `$USER` so you don't have to type your exact username:

```
find /tmp/ -user $USER 2>/dev/null
```